src/libm-benchmarks/plot-scripts/Makefile

```diff
@@ -0,0 +1,110 @@
+BENCHSLEEF := $(shell command -v benchsleef 2> /dev/null)
```

Comment:
Remove, no makefiles.

Reply:
Did you even read the comment at the top of this page?

Reply:
Yes, I still think we shouldn't have makefiles down inside the tree of `src`.

If you really want to keep the makefiles, then I think you should move all this benchmarks and relative scripts at the same level of `src`, in a folder like `svml-benchmarking`, and keep it as a sort of independent project inside SLEEF.

Reply:
Originally, the whole directory including all the C codes were like an independent project.
I think that is easier to understand.
Why do you think C codes have to be compiled by cmake?